notebooks/en/trl_grpo_reasoning_advanced_reward.ipynb

```diff
@@ -0,0 +1,1004 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        remove_unused_columns=False,
</pre><p>this is already false by default, I'd recommend removing for clarity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>